ui/src/app/applications/components/utils.tsx

```diff
@@ -401,6 +402,107 @@ export function renderResourceMenu(
     );
 }
 
+export function renderResourceButtons(
+    resource: ResourceTreeNode,
+    application: appModels.Application,
+    tree: appModels.ApplicationTree,
+    appContext: AppContext,
+    appChanged: BehaviorSubject<appModels.Application>,
+    getApplicationActionMenu: () => any
+): React.ReactNode {
+    let menuItems: Observable<ActionMenuItem[]>;
+
+    if (isAppNode(resource) && resource.name === application.metadata.name) {
```

Comment:
@rbreeze, correct me if I'm wrong but I don't think you are showing any app nodes in this view, so we could probably get rid of this if block?

Reply:
Correct